f8a_report/stack_report_main.py

```diff
@@ -33,8 +44,60 @@ def main():
         logger.exception("Error Generating v2 report")
         raise e
 
-    return response
+    return missing_latest_nodes
 
 
 if __name__ == '__main__':
-    main()
+    missing_latest_nodes = main()
+
+    """
+    Initializing Selinon queues and celery workers.
+
+    If report has packages which are missing any version from Db, ingestion flow
+    for such packages will be triggered to ingest requested version into Graph DB
+    using Jobs API.
+    """
+    if _INVOKE_API_WORKERS == "1":
+        _INGESTION_API_URL = "http://{host}:{port}/{endpoint}".format(
+            host=_INGESTION_SERVICE_HOST,
+            port=_INGESTION_SERVICE_PORT,
+            endpoint=_INGESTION_SERVICE_ENDPOINT)
+
+        # Make ingestion API call for each ecosystem having list packages
+        # and details will be set in dictionary
+        for eco, items in missing_latest_nodes.items():
+
+            input_json = {
+                "flow_arguments": [],
+                "flow_name": "bayesianApiFlow"
```

Comment:
doesnt make any sense to use ingestion api and then call flows etc from outside. Ingestion API should be smart enough to make these decisions. From other services, it should just call the ingestion API call.